apps/api/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:20-alpine as build
+
+WORKDIR /app
+
```

Comment:
Please add the environmental variables in here so that it's easier for the users to understand what's being consumed!

Reply:
Done 👍🏻